code/game/machinery/requests_console.dm

```diff
@@ -455,3 +597,37 @@ var/list/obj/machinery/requests_console/allConsoles = list()
 			msgStamped = "<font color='blue'><b>Stamped with the [T.name]</b></font>"
 			updateUsrDialog()
 	return
+
+
+
+
+
+//{FAX} Code ported over from photocopier.dm
+//Future todo: centralise photocopier functionality
+
+/obj/machinery/requests_console/attackby(obj/item/O, mob/user)

```

Comment:
There is already an attackby() for this machine further up in the document, this code should be merged into that.

https://github.com/genericAnomaly/-tg-station/blob/rc_fax/code/game/machinery/requests_console.dm#L555

And yes, byond will let you compile with two versions of the same proc without warning or sanity.  It even works, in some convoluted way.